quinn/src/udp.rs

```diff
@@ -74,6 +74,8 @@ pub struct RecvMeta {
     pub addr: SocketAddr,
     pub len: usize,
     pub ecn: Option<EcnCodepoint>,
+    /// The destination IP address which was encoded in this datagram
+    pub dest_ip: Option<IpAddr>,
```

Comment:
Let's use `dst` instead of `dest` here, for consistency with similar naming elsewhere.